pkg/drivers/linode/linode.go

```diff
@@ -203,6 +221,24 @@ func (d *Driver) SetConfigFromFlags(flags drivers.DriverOptions) error {
 		return fmt.Errorf("linode driver requires the --linode-root-pass option")
 	}
 
+	stackScript := flags.String("linode-stackscript")
+	if stackScript != "" {
+		ss := strings.SplitN(stackScript, "/", 2)
```

Comment:
`username/label` is somewhat contrived.  I think we should support the numeric StackScript ID, and this format.  If the value passes the Atoi conversion we know it is a StackScript ID, otherwise attempt the "/" splitting.

Reply:
Checking with the docs, I don't see `username` as a filterable field.  Did this work for you? (I'll check if the docs are wrong here).

FWIW, I have suggested that StackScripts should offer a `username/label` identifier.  It could happen in the future.